### ISSUE TITLE
Training loss when using PackedDeformConv does not converge

### ISSUE THREAD

**Issue by omar-ashinawy** [inspiros/tvdcn#7]
I used the packed version of the deformable convolution and the training loss does not converge. I added batch normalization to the output of the offset and mask generators and it worked fine.

The code after adding BatchNorm2d:
`self.offset_generator = nn.Sequential(
                nn.Conv2d(
                self.in_channels,
                2 * self.kernel_size[0] * self.kernel_size[1] * self.offset_groups,
                kernel_size=self.kernel_size,  # type: ignore[arg-type]
                stride=self.stride,  # type: ignore[arg-type]
                padding=self.padding,
                dilation=self.dilation,  # type: ignore[arg-type]
                groups=self.offset_groups,
                bias=generator_bias,
                device=device,
                dtype=dtype),
                nn.BatchNorm2d(2 * self.kernel_size[0] * self.kernel_size[1] * self.offset_groups)
            )`

`self.mask_generator = nn.Sequential(
                nn.Conv2d(
                self.in_channels,
                self.kernel_size[0] * self.kernel_size[1] * self.mask_groups,
                kernel_size=self.kernel_size,  # type: ignore[arg-type]
                stride=self.stride,  # type: ignore[arg-type]
                padding=self.padding,
                dilation=self.dilation,  # type: ignore[arg-type]
                groups=self.mask_groups,
                bias=generator_bias,
                device=device,
                dtype=dtype),
                nn.BatchNorm2d(self.kernel_size[0] * self.kernel_size[1] * self.mask_groups)
            )`
